samples/kvsWebRTCClientViewer.c

```diff
@@ -2,6 +2,31 @@
 
 extern PSampleConfiguration gSampleConfiguration;
 
+// onMessage callback for a message received by the viewer on a data channel
+VOID dataChannelOnMessageCallback(UINT64 customData, PRtcDataChannel pDataChannel, BOOL isBinary, PBYTE pMessage, UINT32 pMessageLen)
+{
+    UNUSED_PARAM(customData);
+    STATUS retStatus = STATUS_SUCCESS;
+    if (isBinary) {
+        DLOGI("DataChannel Binary Message");
+    } else {
+        DLOGI("DataChannel String Message: %.*s\n", pMessageLen, pMessage);
+    }
+}
+
+// onOpen callback for the onOpen event of a viewer created data channel
+VOID dataChannelOnOpenCallback(UINT64 customData, PRtcDataChannel pDataChannel) {
+    DLOGI("New DataChannel has been opened %s \n", pDataChannel->name);
+    dataChannelOnMessage(pDataChannel, customData, dataChannelOnMessageCallback);
+    ATOMIC_INCREMENT((PSIZE_T) customData);
+    STATUS retStatus = STATUS_SUCCESS;
```

Comment:
To adhere with C89 standard variables need to be declared the top of a scope block.  You will need to move the declaration of `retStatus` to the top of this function.

Reply:
Moved the declaration of `retStatus` to the top of this function.